### PR TITLE
new keys and values have been added.

### PR DESCRIPTION
**Pull request by mdhe1248**
"x translation in pixels"
"y translation in pixels"
"rotation angle in degree"

### CODE REVIEW

File: src/ImagineFormat.jl [hunk 1]
```diff
@@ -224,7 +224,11 @@ const field_key_dict = Dict{AbstractString,Function}(
     "hend"                         => x->parse(Int,x),
     "vstart"                       => x->parse(Int,x),
     "vend"                         => x->parse(Int,x),
-    "angle from horizontal (deg)"  => float64_or_empty)
+    "angle from horizontal (deg)"  => float64_or_empty,
+    "x translation in pixels"      => x->parse(Int,x) != 0,
+    "y translation in pixels"      => x->parse(Int,x) != 0,
+    "rotation angle in degree"     => x->parse(Int,x) != 0)
```
Comment (Cody-G) [inline src/ImagineFormat.jl]: I think this should be parsed as a Float64.  @kdw503 can you confirm this?

Comment (Cody-G) [reply src/ImagineFormat.jl]: (I'm referring only to the rotation angle)

Comment (kdw503) [reply src/ImagineFormat.jl]: Yes. 'rotation angle in degree' saved as float.

Comment (mdhe1248) [reply src/ImagineFormat.jl]: Okay, 

I changed it and committed again. I will merge the change sometime soon.

**Comment by Cody-G** [comment]
Looks good to me now.  It would be better if you could "squash" together the two commits because the second one is just an edit of the first.  This makes it easier to understand and manage the commit history.  Have you used git rebase before?  You could do `git rebase -i HEAD~3` and then squash the relevant commits.  After that you'll either have to "force push" to this branch on github or create a new branch.  Creating a new branch isn't a bad idea since if you make a mistake you can just go back to the old branch.

**Comment by timholy** [comment]
Github also makes squashing easy if you want to squash all this to a single commit: in the "merge" box, you can select "Squash and merge" from the dropdown.

**Comment by mdhe1248** [comment]
Thank you Cody and Tim, 
I spent some time in studying Git to understand Cody's comment. "-i" might be interactive. "HEAD" might be something like current status/branch. "HEAD~3" might be three commits before the HEAD. Are these correct? I will spend some more time on rebase and squash. I briefly read some document but need some time to digest.

**Comment by Cody-G** [comment]
Yes you've understood it correctly.  HEAD basically means the current / newest revision of the code on the branch currently checked out.

**Pull request merged**